hack/kwok/stages/pod-delete.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kwok.x-k8s.io/v1alpha1
```

Comment:
Why put this in the hack and not make this part of the Kwok package? e.g. `kwok/stages` or even make it part of the chart directly? I don't really see this as a hack? Also, is this something that we built or something that's available upstream from kwok?

Reply:
The chart is a better place for this CR, and the CR is something that we built wrote using docs/examples from upstream KWOK